posts/navarroaxel/19.git-rebase-exec.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Git rebase exec to validate every commit
+published: true
+description: How to run unit tests and build tools for every commit when you rebase
+tags: 'git, bash, productivity, tooling'
+cover_image: ./assets/git-rebase-exec.png
+id: 1949938
+---
+
+In the previous post we checked out how the `git rebase` works and how to use the `--interactive` mode to keep your topic branch up-to-date with the target (e.g. `main`). Adding the `exec` command to run the tests and lint tools to check every rebased commit.
```

Comment:
I don't understand if this is an enumeration or why the second sentence uses gerund 🤔

Reply:
```suggestion
In the previous post we checked out how the `git rebase` works and how to use the `--interactive` mode to keep your topic branch up-to-date with the target (e.g. `main`). Also, how to use the `exec` command into the `git-rebase-todo` file to run custom commands for the rebased commits.
```